views/partials/scan_form.hbs

```diff
@@ -1,14 +1,12 @@
   <form action="/scan" class="email-scan" method="post" data-no-csrf>
       <div class="input-group">
-          <input class="input-group-field email-to-hash" type="email" name="email" placeholder="Enter Email Address">
+          <input class="input-group-field email-to-hash" type="email" name="email" placeholder="Enter Email Address" required="required">
```

Comment:
Not pretty, but that's browser validation for you...

![firefox_monitor_required](https://user-images.githubusercontent.com/557895/41179177-a2ffaf6c-6b1e-11e8-8745-bd718444540f.png)